src/main/java/mate/academy/spring/validation/RepeatPassword.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
```

Comment:
it's better to put such annotation above class, because technically you checks two fields instead of one

Reply:
I meant to make it so @RepeatPassword was above the class name, not a specific field, but it's an optional

Reply:
Understood, thanks! I've implemented this changing ElementType to TYPE and placing the annotation @RepeatPassword over the UserRequestDto class